File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/NotaGeneralaPeMembriiArbReportRowModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.math.BigDecimal;

public class NotaGeneralaPeMembriiArbReportRowModel {

	private String banca;
	private String comisie;
	private BigDecimal notaFinalaComisie;
	private Integer rankNotaComisie;

	public String getBanca() {
		return banca;
	}

	public void setBanca(String banca) {
		this.banca = banca;
	}

	public String getComisie() {
		return comisie;
	}

	public void setComisie(String comisie) {
		this.comisie = comisie;
	}

	public BigDecimal getNotaFinalaComisie() {
		return notaFinalaComisie;
	}

	public void setNotaFinalaComisie(BigDecimal notaFinalaComisie) {
		this.notaFinalaComisie = notaFinalaComisie;
	}

	public Integer getRankNotaComisie() {
		return rankNotaComisie;
	}

	public void setRankNotaComisie(Integer rankNotaComisie) {
		this.rankNotaComisie = rankNotaComisie;
	}

}

File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/nota-generala-pe-membrii-arb-report-row.model.ts
import { JsonObject, JsonProperty } from "json2typescript";

@JsonObject
export class NotaGeneralaPeMembriiArbReportRowModel {

	@JsonProperty("banca", String)
    public banca: string = null;

	@JsonProperty("comisie", String)
    public comisie: string = null;

	@JsonProperty("notaFinalaComisie", Number)
    public notaFinalaComisie: number = null;

	@JsonProperty("rankNotaComisie", Number)
    public rankNotaComisie: number = null;
 
}